WordPress/src/test/java/org/wordpress/android/ui/posts/editor/EditorActionsProviderTest.kt

```diff
@@ -0,0 +1,168 @@
+package org.wordpress.android.ui.posts.editor
+
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.times
+import com.nhaarman.mockitokotlin2.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers.any
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.fluxc.model.post.PostStatus
+import org.wordpress.android.util.CrashLoggingUtilsWrapper
+
+@RunWith(MockitoJUnitRunner::class)
+class EditorActionsProviderTest {
+    private var crashLoggingUtilsWrapper: CrashLoggingUtilsWrapper = mock()
```

Comment:
Is there a reason why this wrapper has to be mutable? If not then we could utilize `val` here. Let me know your thoughts 👍

Reply:
No reason - I had `setUp` method but since I removed it, it's better to use `val` now;). Fixed in 2bd574b13e755a27c0ad280ed27305f757841657